src/OpenTelemetry.AspNetCore/OpenTelemetryServicesExtensions.cs

```diff
@@ -0,0 +1,86 @@
+﻿// <copyright file="OpenTelemetryServicesExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+
+using OpenTelemetry.Instrumentation.AspNetCore;
+using OpenTelemetry.Instrumentation.Dependencies;
+using OpenTelemetry.Trace.Configuration;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for setting up OpenTelemetry services in an <see cref="IServiceCollection" />.
+    /// </summary>
+    public static class OpenTelemetryServicesExtensions
+    {
+        /// <summary>
+        /// Adds OpenTelemetry services to the specified <see cref="IServiceCollection" /> with automatic data collection.
```

Comment:
nit: similar suggestion applies to next function
```suggestion
        /// Adds OpenTelemetry to the specified <see cref="IServiceCollection" /> with automatic data collection.
```